src/pages/Homepage/HomepageStatictics.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+	<div class="small-10 small-offset-1 large-10 large-offset-1 columns stats-box">
+		<div class="row">
+			<!-- loans funded -->
+			<div class="small-6 large-2 large-offset-1 columns stats-box__group even">
+				<div class="stats-box__stat">
+					${{ amountFunded | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					loans funded
+				</div>
+			</div>
+			<!-- # lenders  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numLenders | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					lenders
+				</div>
+			</div>
+			<!-- repayment rate -->
+			<div class="small-6 large-2 columns stats-box__group even">
+				<div class="stats-box__stat">
+					{{ repaymentRate | numeral('0%') }}
+				</div>
+				<div class="stats-box__stat-label">
+					repayment rate
+				</div>
+			</div>
+			<!-- # countries  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numCountries }}
+				</div>
+				<div class="stats-box__stat-label">
+					countries
+				</div>
+			</div>
+			<!-- # years -->
+			<div class="small-12 large-2 columns">
+				<div class="stats-box__stat">
+					{{ yearsSinceStartCalc }}
+				</div>
+				<div class="stats-box__stat-label">
+					years
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import { differenceInYears } from 'date-fns';
+import homepageStatistics from '@/graphql/query/homepage/homepageStatistics.graphql';
+
+export default {
+	inject: ['apollo', 'federation'],
+	apollo: {
+		preFetch(config, client) {
+			return client.query({
+				query: homepageStatistics,
+			}).then(({ data }) => {
+				console.log('data', data);
+			});
+		},
+	},
+	created() {
+		try {
+			const statisticData = this.apollo.readQuery({
+				query: homepageStatistics,
+			});
+			this.amountFunded = _get(statisticData, 'general.kivaStats.amountFunded');
```

Comment:
@BoulderBrains 
Seeing that you have no variables in your prefetch you could simplify your prefetch setup based on the structure in BraintreeCheckout.vue. https://github.com/kiva/ui/blob/master/src/components/Checkout/BraintreeCheckout.vue#L248

The main takeaway is that you can use a `result` block as the second part of the prefetch function and assign all of this data there, similar to CheckoutPage.vue prefetch setup. Then you can get rid of this entire created block.

Reply:
Thank you @mcstover, I fixed that up in my latest commit.